Dockerfile

```diff
@@ -9,8 +9,12 @@ RUN apk --update upgrade && \
 
 COPY . /tmp/elastalert
 
-RUN mkdir -p /opt/elastalert && \
-    cd /tmp/elastalert && \
+
+RUN mkdir -p /opt/elastalert
+
+COPY . /tmp/elastalert
```

Comment:
This line duplicates line 10. I suggest reverting all your changes to Dockerfile since they are no longer needed.

Reply:
All gone now.